addon/components/chart-component.js

```diff
@@ -48,7 +48,7 @@ export default Ember.Component.extend( ColorableMixin, ResizeHandlerMixin, {
 
   // Transform the view commonly displaced by the margin
   transformViewport: Ember.computed('marginLeft', 'marginTop', function() {
-    return ('translate(' + this.get('marginLeft') + ',' + this.get('marginTop') + ')');
+    return 'translate(' + this.get('marginLeft') + ',' + this.get('marginTop') + ')';
```

Comment:
Can you use template strings here? https://github.com/lukehoban/es6features#template-strings